truss/constants.py

```diff
@@ -98,9 +98,10 @@
 
 REGISTRY_BUILD_SECRET_PREFIX = "DOCKER_REGISTRY_"
 
-TRTLLM_BASE_IMAGE = "baseten/triton_trt_llm:v0.9.0"
+TRTLLM_BASE_IMAGE_AND_EXECUTABLE = ["baseten/triton_trt_llm:v0.9.0", "/usr/bin/python3"]
```

Comment:
nit: I think it's a bit cleaner if we use 2 constants instead of this array
```suggestion
TRTLLM_BASE_IMAGE = "baseten/triton_trt_llm:v0.9.0"
TRTLLM_EXECUTABLE = "/usr/bin/python3"
```